### DISCUSSION TITLE
Welcome to marcel Discussions!

### DISCUSSION THREAD

**Discussion by geophile** [geophile/marcel#9] [General]
## 👋 Welcome!

Hi, and welcome to discussions of marcel. If you are using marcel, I'm interested in your experiences with it -- what do you like? Dislike? What doesn't work?

If you are lurking, and haven't tried marcel, I hope you will change that. I'm not at the point of using it as my regular shell, but it does make a number of things easier. And when I hit a roadblock I try to remove it.

If you are looking to get involved with an open source project, I would love to have some help: planning and building features, fixing bugs, porting, documentation, you name it.

**Comment by AnLAdmin** [comment]
Nice.
